app/src/main/java/com/orange/ouds/app/ui/utilities/composable/CustomizationBottomSheet.kt

```diff
@@ -67,6 +73,14 @@ fun CustomizationBottomSheetScaffold(
         SheetValue.Hidden, SheetValue.PartiallyExpanded -> stringResource(R.string.app_common_bottomSheetCollapsed_a11y)
         SheetValue.Expanded -> stringResource(R.string.app_common_bottomSheetExpanded_a11y)
     }
+    val screenHeight = LocalConfiguration.current.screenHeightDp
+    val customizationContentHeight by animateDpAsState(
+        when (bottomSheetScaffoldState.bottomSheetState.currentValue) {
```

Comment:
Maybe I missed an edge case but it seems that setting `customizationContentHeight` to `screenHeight.dp / 2 - BottomSheetDefaults.SheetPeekHeight` in all cases works the same.